toolchain/binutils/patches/2.40/001-Re-enable-development.-Update-version-to-2.40.0.patch

```diff
@@ -0,0 +1,28 @@
+From 78eca0130feecb321bb6943045b2161583f61dd2 Mon Sep 17 00:00:00 2001
+From: Nick Clifton <nickc@redhat.com>
+Date: Sat, 14 Jan 2023 15:34:17 +0000
+Subject: [PATCH 01/50] Re-enable development.  Update version to 2.40.0
+
+---
+ bfd/development.sh | 2 +-
+ bfd/version.m4     | 2 +-
+ gas/doc/.dirstamp  | 0
+ 3 files changed, 2 insertions(+), 2 deletions(-)
+ delete mode 100644 gas/doc/.dirstamp
+
+--- a/bfd/development.sh
++++ b/bfd/development.sh
+@@ -16,7 +16,7 @@
+ # along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ 
+ # Controls whether to enable development-mode features by default.
+-development=false
++development=true
```

Comment:
I should probably remove this patch too.

Reply:
Yes, you should.

Reply:
If follow up patches don't depend on it, the translation ones too, maybe?

Reply:
@dhewg I don't think it is necessary.